Dockerfile

```diff
@@ -3,9 +3,7 @@ FROM python:3.12-bookworm AS dev
 RUN apt-get update && \
     apt-get install build-essential debhelper devscripts \
     equivs dh-virtualenv python3-venv dh-sysuser dh-exec \
-    -y --no-install-recommends && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+    -y --no-install-recommends
```

Comment:
Why we do not clean the cache any longer?

Reply:
Mk-build-deps (make debian-deps) will do again apt-get update and install other dependencies. Furthermore this is a dev image so I'd say it's ok to not clean it.